src/main/java/com/ss/training/utopia/counter/entity/Flight.java

```diff
@@ -17,84 +17,87 @@
 @IdClass(FlightPk.class)
 public class Flight implements Serializable {
 
-	private static final long serialVersionUID = -8418526716923540697L;
+	private static final long serialVersionUID = -9104220295315031366L;
 
 	@Id
 	@Column
-	private int departId, arriveId;
+	private Long departId, arriveId;
```

Comment:
Why make these longs instead of Integers?

Reply:
To support the largest possible range of ID values.

More specifically, Java's Integer and MySQL's INT use the same number of bits for integers, but Java's are signed and MySQL's are unsigned, so the maximum value in Java is half that in MySQL.